### PR TITLE
Improve csproj sanity

### PR DESCRIPTION
**Pull request by Therzok**
Move common properties into their own props/target file.

Use globs to include sources.

### CODE REVIEW

**Comment by Therzok** [comment]
Don't merge until big PRs are merged

**Comment by Therzok** [comment]
Also, want to migrate to SDK projects as a last step.

**Comment by Therzok** [comment]
I can actually reduce the huge move.

**Review by hubastard — APPROVED** [review]

File: PluginImports.targets
Comment (Mushroom) [inline PluginImports.targets:7]: What exactly is this? I couldn't find a use of this anywhere and google doesn't give me much to go on

**Review by Mushroom — CHANGES_REQUESTED** [review]
Just the one comment - marking this as changes requested so nobody jumps the gun and merges this yet, as we should probably wait until at least the ILS changes are in

File: PluginImports.targets
Comment (Therzok) [reply PluginImports.targets:7]: Oh, it's stuff I haven't committed yet, my bad.

Want to use the prop in the copy to game dir targets.

Comment (Therzok) [reply PluginImports.targets]: Fixed.

**Comment by Therzok** [comment]
Reorg'd things a bit.

**Review by hubastard — APPROVED** [review]

**Comment by Therzok** [comment]
Should be good to merge.

**Comment by Therzok** [comment]
This is getting merge conflicts constantly :D

**Review by Baldie-dev — APPROVED** [review]

**Review by Mushroom — APPROVED** [review]

**Pull request merged**
